### DISCUSSION TITLE
[Help]: Active pseudo class doesn't work with embla

### DISCUSSION THREAD

**Discussion by sedlukha** [davidjerleke/embla-carousel#338] [Help] [answered]
Hello everyone! 

This is not the embla-carousel issue, but I found something strange.

I try to add "active" pseudo-class to the card element. Everything works fine in chrome and safari, but not in firefox.
Here is sandbox example:
https://codesandbox.io/s/embla-carousel-default-react-forked-ipedsy?file=/src/css/embla.css

Maybe someone knows the reason for this problem? Or maybe there is a way to fix (hack) this issue?

And huge thanks to @davidjerleke for this awesome carousel!

**Comment by davidjerleke** [comment, marked as answer]
Hi @sedlukha,

Thanks for reporting this 👍. It seems like Firefox is having issues with `:active` together with `event.preventDefault()` that Embla is invoking when dragging the carousel. This seems to [be a known bug in Firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=771241). Scroll down to the last comment and you'll see that others have encountered the same problem. I'm afraid that the Firefox team has to solve this on their end. 

Best,
David

**Reply by sedlukha** [reply]
Thanks for the answer, @davidjerleke!
Such an old bug in FF!
